astroplan/core.py

```diff
@@ -73,6 +75,80 @@ def _generate_24hr_grid(t0, start, end, N, for_deriv=False):
 
     return t0 + time_grid
 
+# Observatory database and list of names:
+_site_db = None
+_site_names = []
+
+def _load_sites():
+    '''
+    Load observatory database from astroplan/data/observatories.json
+    '''
+    global _site_db, _site_names
+    _site_db = dict()
+    db = json.loads(get_pkg_data_contents('data/observatories.json'))
+    _site_names = [db[site]['name'] for site in db]
+    for site in db:
+        location = EarthLocation.from_geodetic(db[site]['longitude'],
+                                   db[site]['latitude'],
+                                   db[site]['elevation'])
+        _site_db[db[site]['name']] = location
+        for alias in db[site]['aliases']:
+            _site_db[alias.lower()] = location
+
+def get_site(site_name):
+    '''
+    Construct an `~astropy.coordinates.EarthLocation` object for a known
+    observatory.
```

Comment:
"a known observatory" -> "an observatory in the builtin list."?   

Also, perhaps "construct"->"return"?  "construct" implies to me it's a new object every time, but in this case it's actually the *same* object if you ask for the same name twice.  A subtle distinction, but important if someone wants to do ``is`` comparison or the like.